lib/util/array.ts

```diff
@@ -0,0 +1,8 @@
+import is from '@sindresorhus/is';
+
+export function coerceArray<T>(input: T[] | null | undefined): T[] {
+  if (is.array(input)) {
```

Comment:
we should add another check for non-array type and log a debug message when it's not null or undefined.